src/content/core/user-roles-and-permissions.mdx

```diff
@@ -0,0 +1,127 @@
+---
+title: User Roles and Permissions in Okteto
+description: Use volume snapshots in your development environments
+sidebar_label: User Roles and Permissions
+id: user-roles-and-permissions
+---
+
+
+The Okteto platform offers a robust Role-Based Access Control (RBAC) system to manage and control access to different environments within the platform.
+This page outlines the roles, permissions, and key features available to Admins and Developers in Okteto.
+
+## User Roles
+
+In Okteto, there are two primary roles: **Admins** and **Developers**. These roles were previously known as “Admin” and “User,” but we have updated the terminology for clarity and consistency across our platform.
+
+## Admins
+
+Admins have extensive control and oversight capabilities across the Okteto platform. Key features include:
+
+### Admin Access to Any Namespace
+Admins can access any Namespace, allowing them to:
+- Deploy and remove Development Environments
+- Redeploy Development Environments
+- Remove Namespaces entirely
+
+**Benefits**: This level of access allows admins to manage and troubleshoot environments, even if the original owner is unavailable.
+
+### Transferring Namespace Ownership
+Admins can [reassign the owner of a non-personal namespace](core/namespaces.mdx#transfer-namespace-ownership) to another user.
+
+**Benefits**: This provides flexibility in managing user and resource changes, such as team restructuring or personnel off time.
+
+### Full Control Over Environments
+Admins can perform all operations on any Namespace or Preview Environment, including:
+- Deleting, sharing, waking, sleeping, and marking namespaces/previews as persistent
+- Deploying or deleting applications using both Okteto and `kubectl`
+- Accessing all resources and sub-resources within any Development Environment
+
+## Developers
+
+Developers, previously known as “Users,” have permissions primarily focused on creating and managing their own environments. Key features include:
+
+### Creating and Managing Environments
+Developers can:
+- Create Namespaces and Personal/Global Preview Environments
+- Delete Namespaces and Preview Environments they own
+- Share Namespaces and Preview Environments they own
+- Deploy applications using both Okteto and `kubectl` within their own or shared Namespaces and Previews Environments
+- Wake and sleep namespaces and previews they own or have access to
+
+### Access to Global Previews
+Developers have read access to Global Preview Environments, enabling them to troubleshoot issues effectively using `kubectl`.
```

Comment:
`kubectl`, the Okteto UI or the Okteto CLI.
Also, should we mention how to fine tune this level of access?
https://github.com/okteto/docs/pull/772/files#diff-5246fa871de346de64f5cbaf3735699dde71a326365ff143ce7219a80f38f839R897